R-package/R/lgb.Dataset.R

```diff
@@ -998,7 +998,11 @@ slice.lgb.Dataset <- function(dataset, idxset, ...) {
 #' \itemize{
 #'     \item \code{label}: label lightgbm learn from ;
 #'     \item \code{weight}: to do a weight rescale ;
-#'     \item \code{group}: group size ;
+#'     \item{\code{group}: used for learning-to-rank tasks. An integer vector describing how to
+#'         group rows together as ordered results from the same set of candidate results to be ranked.
+#'         For example, if you have a 100-document dataset with \code{group = c(10, 20, 40, 10, 10)},
```

Comment:
In your vector `sum(c(10, 20, 40, 10, 10)) != 100`.

Also please add info about condition `sum(group) = n_samples.` from Python here.

Reply:
ha! Promise I do know how to do simple addition 😆 

fixed in https://github.com/microsoft/LightGBM/pull/3772/commits/d4816468557654f803aef64bcc9e74205d025612